drivers/gles2/rasterizer_gles2.cpp

```diff
@@ -462,6 +462,11 @@ void RasterizerGLES2::end_frame(bool p_swap_buffers) {
 	} else {
 		glFinish();
 	}
+
+	bool multiple_buffer_batching = GLOBAL_GET("rendering/batching/options/multiple_buffer_batching");
+	if (multiple_buffer_batching) {
```

Comment:
`GLOBAL_GET` should not be called every frame, you can load in constructor / initialize and keep a member variable.